src/api/config.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+import { LOCAL_STORAGE_KEYS } from '../constants/localStorageKeys'
```

Comment:
imports style guide issue. fix at all places.

issue: space line between lib and local import missing